apps/web/src/types/index.ts

```diff
@@ -24,6 +24,19 @@ export enum Severity {
   Severe = 'Severe',
 }
 
+export enum VaccinationStatus {
+  None = 'None',
+  Partial2Dose = 'Partial2Dose',
+  Partial1Dose = 'Partial1Dose',
+  Full = 'Full',
+}
+
+export enum AgeRanges {
+  Under50 = 'Under50',
+  FiftyTo69 = 'FiftyTo69',
```

Comment:
Nit, these values are fairly awkward key/value pairs. It might be good to change them to be all letters or numbers

Reply:
I had some issues with these! Mostly involving numbers but also a '+' character broke the tests. I'll change them to all letters.